.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
You can't put comments in JSON

Reply:
Those comments were auto-generated but I can remove them

Reply:
VS: Code always using 'JSONC', which allows comments and trailing commas.  Can keep the comments in if they're useful.